utilities/check_account/checks/check_account_spec.rb

```diff
@@ -38,8 +41,10 @@
       expect(iam_utils.get_users.find { |x| x[:has_password] }).to be_nil
     end
 
-    it "should not have any accesses keys greater than #{MAX_KEY_AGE} days old" do
-      expect(iam_utils.get_active_keys_older_than_n_days(MAX_KEY_AGE)).to be_empty
+    it "any access keys older than #{MAX_KEY_AGE} days should be whitelisted" do
```

Comment:
Rephrase to be a little less encouraging - "should not have any keys greater than #{MAX_KEY_AGE} days old unless explicitly whitelisted"